images_to_samples.py

```diff
@@ -54,6 +53,18 @@ def mask_image(arrayA, arrayB):
     return ma_array
 
 
+def pad_diff(arr, w, h, arr_shape):
```

Comment:
Would be a good habit (for all of us!) to add docstrings to our functions.

Reply:
Good Idea!